test/spinxsk/spinxsk.c

```diff
@@ -398,6 +398,10 @@ AttachXdpProgram(
         flags |= XDP_CREATE_PROGRAM_FLAG_NATIVE;
     }
 
+    // if (RandUlong() % 8) {
+    //     flags |= XDP_CREATE_PROGRAM_FLAG_ALL_QUEUES;
```

Comment:
Was this finding crashes or breaking spinxsk logic?

Reply:
Nope, I accidentally checked this in.